Assets/Scripts/Faction.cs

```diff
@@ -185,8 +263,49 @@ public void ChangeHappiness(float amount)
         }
     }
 
+    ///<summary>
+    /// awards mana to the player and the ai. mana = happiness * power
+    /// </summary>
+    public void AwardMana()
+    {
+        //if the player is happy, award the player mana
```

Comment:
This is a little confusing - reads like it's depending on how happy the player is rather than how happy the faction is with the player